packages/flutter/lib/src/material/popup_menu.dart

```diff
@@ -92,8 +91,9 @@ class _PopupMenuItemState<T extends PopupMenuItem<dynamic>> extends State<T> {
 
     Widget item = new DefaultTextStyle(
       style: style,
-      child: new Baseline(
-        baseline: config.height - _kBaselineOffsetFromBottom,
+      child: new Align(
+        widthFactor: 1.0,
+        alignment: FractionalOffset.centerLeft,
```

Comment:
why is this better? The spec for positioning menus explicitly says that the baseline is to be 20dp from the bottom of the box: https://www.google.com/design/spec/components/menus.html#menus-specs